src/components/snapshots/useSnapshot.ts

```diff
@@ -63,42 +60,68 @@ const useSnapshot: SnapshotOperation = async ({
       throw `Snapshot was created with version: ${snapshotVersion}\n You can't install a snapshot created with a version newer than the current application version: ${config.version}`
     }
 
-    const snapshotObject = JSON.parse(snapshotRaw)
-
-    if (options.shouldReInitialise) {
-      await initialiseDatabase(options, snapshotFolder)
-    }
-
-    if (options.resetFiles) {
+    if (options.resetFiles || options.usePgDump) {
       execSync(`rm -rf ${FILES_FOLDER}/*`)
     }
 
-    // Prevent triggers from running while we insert data, but only for full re-init
-    if (options.shouldReInitialise) {
+    if (options.usePgDump) {
+      // The quick way, using pg_restore -- whole database only
```

Comment:
I'm a bit confused by `whole database only` comment

Reply:
As in, you can't do partial exports (like templates only, or certain tables) like you can using the JSON object -- it's the whole database or nothing. (For now -- it's possible we could do partial export/import with pg_dump, but it'd get very messy and complicated, so no point right now)

Reply:
Could you rephrase the comment though? It's not clear

Reply:
Okay, I'm not really sure what's not clear about it. Hopefully this is better: c7fb4011

Reply:
Hahah it's not that different to be honest. I think if you had added something as you explained in the chat of this review it would have been nice. But anyway, it's not a big deal